pkg/registry/bundle.go

```diff
@@ -117,12 +128,25 @@ func (b *Bundle) ProvidedAPIs() (map[APIKey]struct{}, error) {
 	if err != nil {
 		return nil, err
 	}
-	for _, crd := range crds {
-		for _, v := range crd.Spec.Versions {
-			provided[APIKey{Group: crd.Spec.Group, Version: v.Name, Kind: crd.Spec.Names.Kind, Plural: crd.Spec.Names.Plural}] = struct{}{}
-		}
-		if crd.Spec.Version != "" {
-			provided[APIKey{Group: crd.Spec.Group, Version: crd.Spec.Version, Kind: crd.Spec.Names.Kind, Plural: crd.Spec.Names.Plural}] = struct{}{}
+
+	for _, c := range crds {
+		crd, ok := c.(*apiextensionsv1.CustomResourceDefinition)
```

Comment:
I wonder if we can use a switch statement here like this (https://github.com/operator-framework/operator-registry/blob/master/pkg/lib/bundle/validate.go#L215) so that we can support newer version of CRD in the future.

Reply:
v1 is considered stable, and probably won't change for a very long time, but I think the code does look cleaner this way so lets change it

Reply:
I would consider the switch statement here is cleaner as you can get rid of the `ok`. The `ok` check in else can be translated into `default` statement. On the other note, you can get rid of `continue` line in #138 as it doesn't do anything.